javascript/src/main/pages/TutorAssignment/TutorAssignments.js

```diff
@@ -49,6 +72,7 @@ const TutorAssignment = () => {
     <>
       {isInstructor && newTutorAssignmentButton}
       <TutorAssignmentTable tutorAssignments={tutorAssignmentList} isInstructor={isInstructor}/>
+      <Button><CSVLink style={{color: "white"}} headers={headers} data={tutorAssignmentList} filename = {"TutorAssignments.csv"}>Download CSV</CSVLink></Button>
```

Comment:
As the tutor assignments page increases in size, the download button is harder to find. Would we be able to place the download button above the table?